### PR TITLE
goreleaser: add completion

### PR DESCRIPTION
**Pull request by mchlumsky**
#### Testing the changes
- I tested the changes in this PR: **YES**

- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/goreleaser/template [hunk 1]
```diff
@@ -13,5 +13,14 @@ distfiles="https://github.com/goreleaser/goreleaser/archive/refs/tags/v$version.
 checksum=60e94d9371f7106878441deb006814339e3cf3d7f574d25cb22a9a45ec14939b
 
 post_install() {
+	vdoc README.md
```
Comment (classabbyamp) [inline srcpkgs/goreleaser/template:16]: don't think this is useful, the only documentation-ish thing in that readme is

image

which users already can see with `xbps-query -p homepage goreleaser`

Comment (mchlumsky) [reply srcpkgs/goreleaser/template]: Understood. I dropped it.

File: srcpkgs/goreleaser/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'goreleaser'
 pkgname=goreleaser
 version=1.19.2
-revision=1
+revision=2
 build_style=go
```
Comment (classabbyamp) [inline srcpkgs/goreleaser/template:5]:
```suggestion
build_style=go
build_helper="qemu"
```

File: srcpkgs/goreleaser/template [hunk 3]
```diff
@@ -14,4 +14,11 @@ checksum=60e94d9371f7106878441deb006814339e3cf3d7f574d25cb22a9a45ec14939b
 
 post_install() {
 	vlicense LICENSE.md
+
+	goreleaser=$(find $GOPATH/bin -name goreleaser)
+
+	for shell in bash fish zsh; do
+		${goreleaser} completion ${shell} > completion.${shell}
```
Comment (classabbyamp) [inline srcpkgs/goreleaser/template:21]:
```suggestion
		vtargetrun ${goreleaser} completion ${shell} > completion.${shell}
```

**Review by classabbyamp — COMMENTED** [review]
this should fix cross

File: srcpkgs/goreleaser/template (hunk 3 above)
Comment (mchlumsky) [reply srcpkgs/goreleaser/template]: That fixed it. Thank you!

**Pull request merged**